src/config/product.rosa.js

```diff
@@ -250,23 +250,23 @@ const config = {
   initialInventoryFilters: [
     {
       metric: INVENTORY_TYPES.DISPLAY_NAME,
-      cell: ({ [INVENTORY_TYPES.DISPLAY_NAME]: displayName, [INVENTORY_TYPES.INSTANCE_ID]: instanceId }) => {
+      cell: ({ [INVENTORY_TYPES.DISPLAY_NAME]: displayName, [INVENTORY_TYPES.INVENTORY_ID]: inventoryId }) => {
```

Comment:
Wasn't exactly this change reverted recently?
I was under the impression, that ROSA does not have `inventory_id` in the API response. 
```
{
  'id': 'f3ac9a69-494d-4789-8320-f24f40cab126',
  'instance_id': 'c0ef724a-3dd3-4452-820a-88c086c9ad79',
  'display_name': 'automation_moa-hostedcontrolplane_cluster_c0ef724a-3dd3-4452-820a-88c086c9ad79',
  'billing_provider': 'aws',
  'billing_account_id': 'curiosity-automation-user',
  'measurements': [1.0, 1.0],
  'last_seen': datetime.datetime(2024, 5, 3, 23, 0, tzinfo=tzutc()),
  'last_applied_event_record_date': datetime.datetime(2024, 5, 4, 0, 55, 39, 522964, tzinfo=tzutc()),
  'number_of_guests': 0,
  '_labeled_measurements': {'Cores': 1.0, 'Instance-hours': 1.0}
}
```

Reply:
This work needs to be rebased severely. This highlights how long ago it was completed, before the reversion/fix we did. We'll rebase it, it should eliminate the other comments. [If you look at the actual commit instead of the PR/MR you'll see the limited work here](https://github.com/RedHatInsights/curiosity-frontend/pull/1305/commits/4173dda38e51187b4adc2eaec03e7fa8db400cfb)